### ISSUE TITLE
Changing the github username on a boxen install

### ISSUE THREAD

**Issue by adamnbowen** [boxen/our-boxen#608]
I just set up a new coworker's machine with boxen.  To speed things up, I associated it with my own username, assuming I could switch over to his github username once he arrived.  However, I can't seen to figure out a way to switch boxen over to his new username—I've edited both /opt/boxen/config/boxen/defaults.json and /opt/boxen/env.d/30_gh_creds.sh and then run `source /opt/boxen/env.sh`. Running `facter` inside the `/opt/boxen/repo` directory at this point shows the new github username, but once I run `boxen`, it just overwrites the new username with the old one.

Is there any way to change the github username without running `./script/nuke --all --force` and then reinstalling boxen?

**Comment by PhilAWilks** [comment]
I made the mistake of creating a github username with a dash (-).  I edited it to remove the dash, but then found that the boxen script (/tmp/install-pboxen) wasn't being provided with the new user - it always contained the original username along with the original token.  I couldn't find any way to force the script to pick up the new username, so I did the following in order to get Boxen installed:

- Logged into github with my new username (no dashes) and regenerated my token in [Personal Settings -> Applications -> Personal Access Tokens -> boxen -> Edit -> Regenerate Token]
- Make a note of the new token value
- curl -s https://bitnet-boxen-web.herokuapp.com/script/<nnnnnn>.sh > /tmp/install-boxen (to get my install script without actually running it)
- Edited the script (/tmp/install-boxen) to contain the correct username and access token on the following two lines:

1) curl --progress-bar -L 'https://api.github.com/repos/org123/org123-boxen123/tarball/master?access_token=<NEW TOKEN>' | tar -xz - --strip-components 1 && touch .snapshot
2) script/boxen --token <NEW TOKEN> --login <NEW USERNAME>

- Run the script using: bash /tmp/install-boxen

This worked for my purposes - it's very possible there's a better way of having the updated username and access tokens being automatically pushed to the boxen script... I'm very much a newbie to this world.
